src/gfx/modes/groups/BondsGroup.js

```diff
@@ -7,65 +7,76 @@ function getCylinderCount(bondOrder) {
   return bondOrder < 2 ? 1 : bondOrder;
 }
 
-function BondsGroup(geoParams, selection, colorer, mode, transforms, polyComplexity, material) {
-  var drawMultiple = mode.drawMultiorderBonds();
-  var showAromatic = mode.showAromaticLoops();
-  var bondsIdc  = selection.chunks;
-  var bonds = selection.bonds;
-  var bondsCount = 1;
-  for (var i = 0, n = bondsIdc.length; i < n; ++i) {
-    bondsCount += this.getBondOrder(bonds[bondsIdc[i]], drawMultiple, showAromatic);
+class BondsGroup extends ChemGroup {
+  constructor(geoParams, selection, colorer, mode, transforms, polyComplexity, material) {
+    super(geoParams, selection, colorer, mode, transforms, polyComplexity, material);
+    const drawMultiple = mode.drawMultiorderBonds();
+    const showAromatic = mode.showAromaticLoops();
+    const bondsIdc = selection.chunks;
+    const bonds = selection.bonds;
+    let bondsCount = 1;
+    for (let i = 0, n = bondsIdc.length; i < n; ++i) {
+      bondsCount += this.getBondOrder(bonds[bondsIdc[i]], drawMultiple, showAromatic);
+    }
+    this._geoArgs = [bondsCount, polyComplexity];
   }
-  this._geoArgs = [bondsCount, polyComplexity];
-  ChemGroup.call(this, geoParams, selection, colorer, mode, transforms, polyComplexity, material);
```

Comment:
This is not an equivalent transform: the parent constructor was called at the end, now it is called as the top of the function. That's why the viewer crashes. Please try to fix the issue (however, it's no so easy to do) and look for similar issues in other files.